From b5e02ceafa810ceeb55f6c1f08d7b111ceef72f8 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Sun, 17 Apr 2022 12:59:18 +0300
Subject: [PATCH] [SW-0] gaudi3: update cpu frequency macro

Currently, the ARC cpu frequency is set to 500MHz. In gaudi3, the ARC
cpu POR frequency is 1000MHz (1GHz), so update macro accordingly.

Change-Id: Iefda01f72b8273d7230aba983a8a0e0efb86b2db
---

diff --git a/arch/gaudi3/arc/soc.h b/arch/gaudi3/arc/soc.h
index 8bc9c50..24fe6fc 100644
--- a/arch/gaudi3/arc/soc.h
+++ b/arch/gaudi3/arc/soc.h
@@ -26,10 +26,8 @@
 	REGION_LBW_CFG_IDX = 15,
 };
 
-/* TODO: Replace with gaudi3 values */
-
-#define CPU_FREQ			500000000
-#define CPU_FREQ_US			500	/* CPU_FREQ / 1000000 */
+#define CPU_FREQ			1000000000
+#define CPU_FREQ_US			1000	/* CPU_FREQ / 1000000 */
 
 /* TODO: Remove unused defines */
 #define FW_SIZE				65535
